ci/rt.sh

```diff
@@ -283,8 +317,29 @@ if [ "$build_exe" == "yes" ]; then
   postmsg "$logfile" "$msg"
 fi
 
-#Setting tests
-export test_list="sfs gefsv12 gefsv13 nmmb rap hrrr hafs 3drtma mpas mpas_hfip rrfs rrfs_ifi_missing gfs"
+# Create job cards from template for RDHPCS
+if [[ ${machine} != "wcoss2" ]]; then
+   
+   cd $svndir/ci/jobs-dev
+
+   source machine.sh
+   source test.sh
+   source atparse.sh
+
+   for test in ${test_list}
+   do
+      set_global
+      if [[ ${test} =~ "gefs" ]]; then
```

Comment:
@gspetro-NOAA Do the 'gefs' and 'rtma' tests have specific configurations different from the other tests? Could this IF block be combined as "$test}"?